docs/exchanges.md

```diff
@@ -0,0 +1,22 @@
+# Exchanges
+
+## Persisted Queries
```

Comment:
Can we maybe structure this as an "awesome" list? Short link and description on a single line?

I know we don't have much content yet, but it'll probably scale better 👍